test/potluck_test.rb

```diff
@@ -0,0 +1,48 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "pry"
+require "./lib/potluck"
+
+class PotluckTest < Minitest::Test
+  def test_it_exists
+    potluck = Potluck.new("7-13-18")
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_it_has_a_date
+    potluck = Potluck.new("7-13-18")
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_it_has_no_dishes
+    potluck = Potluck.new("7-13-18")
+    assert_equal [], potluck.dishes
+  end
+
+  def test_it_has_couscous
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    assert_instance_of Dish, couscous_salad
+  end
+
+  def test_it_has_meatballs
+    potluck = Potluck.new("7-13-18")
+    cocktail_meatballs = Dish.new("Cocktail Meatballs", :entree)
+    assert_instance_of Dish, cocktail_meatballs
+  end
+
+  def test_adding_couscous_dish_to_the_potluck
+    potluck = Potluck.new("7-13-18")
+    couscous_salad = Dish.new("Couscous Salad", :appetizer)
+    assert_equal couscous_salad, potluck.add_dish(couscous_salad)
```

Comment:
This is a case where I actually don't care that much what `add_dish` returns. What I really care about is how it changes the return value of `potluck.dishes`. I would change this test to something like the following:

```
def test_it_can_add_a_dish
  potluck = Potluck.new("7-13-18")
  couscous_salad = Dish.new("Couscous Salad", :appetizer)

  potluck.add_dish(couscous_salad)

  assert_equal [couscous_salad], potluck.dishes
end
```